torch_frame/data/multi_nested_tensor.py

```diff
@@ -275,6 +275,27 @@ def _single_index_select(self, index: int, dim: int) -> MultiNestedTensor:
         else:
             raise RuntimeError(f"Unsupported dim={dim} for index_select.")
 
+    def fillna_col(
+        self,
+        index: int,
+        fill_value: Union[int, float, Tensor],
+    ) -> None:
+        start_idx = torch.arange(
+            index,
+            self.num_rows * self.num_cols,
+            self.num_cols,
+            device=self.device,
+        )
+        diff = self.offset[start_idx + 1] - self.offset[start_idx]
+        batch, arange = _batched_arange(diff)
+        # Compute values
+        values = self.values[self.offset[start_idx][batch] + arange]
```

Comment:
rename to `values_col` as I understand this extracts `values` with the specified `col_index`.